
class AzeError(Exception):
    pass

class AzeRequestError(AzeError):
    pass
